lib/board.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+
+class Board 
+  attr_accessor :cells
+
+  def initialize(cells = Array.new(9, " "))
+    @cells = cells
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " "
+    puts " "
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    puts " "
+    puts " "
+  end
+
+
+  def update(position, player)
+    @cells[position.to_i - 1] = player.token
+  end
+
+  def position(position)
+    @cells[position.to_i - 1]
+  end
+
+  def taken?(position)
+    !(@cells[position.to_i - 1].nil? || @cells[position.to_i - 1] == " ")
+  end
+
+  def valid_move?(position)
+    if (taken?(position.to_i) == false) && (position.to_i.between?(1,9) == true)
+    true
+    else (taken?(position.to_i) == true) || (position.to_i.between?(1,9) == false)
+    false
+    end
+  end
```

Comment:
Use the implicit boolean return value of the expression instead of adding the `if` statement. 
```
def valid_move?(position)
  (taken?(position.to_i) == false) && (position.to_i.between?(1,9) == true)
end
```
Also, eqaulity comparison to a boolean literal value is never necessary; just use the implicit return value of the expression. Finally, use the bang `!` operator to negate or reverse a logical comparison.
```
def valid_move?(position)
  !taken?(position.to_i) && position.to_i.between?(1,9)
end
```